iceoryx_hoofs/include/iceoryx_hoofs/cxx/.clang-tidy

```diff
@@ -0,0 +1,17 @@
+InheritParentConfig:  true
+
+CheckOptions:
+  - { key: readability-identifier-naming.ClassCase,               value: lower_case }
+  - { key: readability-identifier-naming.EnumCase,                value: lower_case }
+  - { key: readability-identifier-naming.StructCase,              value: lower_case }
+  - { key: readability-identifier-naming.UnionCase,               value: lower_case }
+  - { key: readability-identifier-naming.MethodCase,              value: lower_case }
+  - { key: readability-identifier-naming.FunctionCase,            value: lower_case }
+  - { key: readability-identifier-naming.NamespaceCase,           value: lower_case }
+  - { key: readability-identifier-naming.PrivateMemberPrefix,     value: m_ }
```

Comment:
Is this also valid for `struct`'s? We have excluded `struct`s from this rule in the `CONTRIBUTING.md`.

Reply:
Yes that applies to structs too, afaik Clang-tidy is not able to differentiate between classes or structs when enforcing this rule. For easier handling I would propose to remove this exclusion from the Contributing.md

@elBoberido since this rule came from you, do you agree on that?

Reply:
Sure, but the clang-tidy rule is for private member and should therefore also not affect public member not having a `m_` prefix